example/image_classification_mobilenet/lib/ui/camera.dart

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'dart:io';
+import 'package:camera/camera.dart';
+import 'package:flutter/material.dart';
+import 'package:image_classification_mobilenet/helper/image_classification_helper.dart';
+
+class CameraScreen extends StatefulWidget {
+  const CameraScreen({
+    super.key,
+    required this.camera,
+  });
+
+  final CameraDescription camera;
+
+  @override
+  State<StatefulWidget> createState() => CameraScreenState();
+}
+
+class CameraScreenState extends State<CameraScreen>
+    with WidgetsBindingObserver {
+  late CameraController cameraController;
+  late ImageClassificationHelper imageClassificationHelper;
+  Map<String, int>? classification;
+  bool _isProcessing = false;
+
+  // init camera
+  initCamera() {
+    cameraController = CameraController(widget.camera, ResolutionPreset.medium,
+        imageFormatGroup: Platform.isIOS
+            ? ImageFormatGroup.bgra8888
+            : ImageFormatGroup.yuv420);
+    cameraController.initialize().then((value) {
+      cameraController.startImageStream(imageAnalysis);
+      setState(() {});
```

Comment:
Let's wrap these in a mounted check, otherwise they get called after dispose() due to a race condition:

    if( mounted ) {
      setState(() {});
    }